### ISSUE TITLE
Misleading error message when you use a class that doesn't have an import

### ISSUE THREAD

**Issue by sethladd** [flutter/flutter-intellij#1313]
<img width="758" alt="screen shot 2017-09-03 at 3 55 53 pm" src="https://user-images.githubusercontent.com/5479/30003616-7031b46a-90c0-11e7-9d3c-b12bb0ccdced.png">

Here, I construct a class, one which is not yet imported into my file.

The first error is confusing and probably not helpful: "returns type dynamic" .. ? it's returning a type of LotteryStateBadge

The second error is probably the one we want, however I'd love for it to also say ", not found in any imports"

And then, I'd love for a quick fix to be added right here saying "Import this class?"

Thanks!

**Comment by sethladd** [comment]
Note: IntelliJ does show a little pink lightbulb over on the left, which does has the quick fix to import this class. So, someone somewhere knows this is just a missing import, not "returning a type of dynamic". Maybe we can update these error messages when a quick fix is identified?

**Comment by pq** [comment]
cc @bwilkerson

**Comment by bwilkerson** [comment]
I agree, there should only be one error message. Hopefully we can have that in the new front end.

**Comment by pq** [comment]
Is there a better place we can track this?  @bwilkerson : should we move it to the SDK maybe?

**Comment by bwilkerson** [comment]
I'm not sure whether the front end work is far enough along to be able to know whether it currently has a problem in this area, and it seems a little strange to open a "please make sure this isn't a bug" issue (assuming it isn't far enough along). I'm not sure what the right answer is for tracking this.

**Comment by mit-mit** [comment]
@bwilkerson what is the status of this in the new FE?

**Comment by bwilkerson** [comment]
Sorry, but I don't know. There ought to be a way to run kernel to see error messages, but I don't know what that is.

**Comment by RedBrogdon** [comment]
Bumping this issue, as I'm running into it with method overrides.

If I have these two files:

### file1.dart
```dart
import 'dart:async';

abstract class BaseClass {
  FutureOr<int> someMethod();
}
```

### file2.dart
```dart
import 'file1.dart';
// Note the lack of an import for dart:async needed by FutureOr<>

class SubClass extends BaseClass {
  @override
  FutureOr<int> someMethod() {}
}
```

Analyzer from the command line reports these problems:

    error • 'SubClass.someMethod' ('() → dynamic') isn't a valid override of 'BaseClass.someMethod' ('() → FutureOr<int>') • lib/file2.dart:5:3 • invalid_override
    error • Undefined class 'FutureOr' • lib/file2.dart:6:3 • undefined_class

But IntelliJ only gives you the first one when you hover over the error, which is the first thing most people see. This resulted in an issue for my state management project (https://github.com/RedBrogdon/rebloc/issues/7), so I can offer at least a tiny bit of anecdotal evidence that people are getting confused.
